content/docs/command-reference/exp/pull.md

```diff
@@ -62,9 +63,9 @@ used.
 - `-r <name>`, `--remote <name>` - name of the `dvc remote` to pull cached files
   from.
 
-- `--run-cache` - downloads all available history of [stage runs] from the
-  `dvc remote` (to the cache only, like `dvc fetch --run-cache`). Note that
-  `dvc exp run <stage_name>` is necessary to checkout these files.
+- `--run-cache`, `--no-run-cache` - downloads all available history of [stage
```

Comment:
@dberenbaum just to clarify, what is the current status after all? do we push or not by default (I assume no)? why do have these options togethers with the same description - is it because `no-run-cache` doesn't do anything?

Reply:
Correct

Reply:
so, if I specify `--no-run-cache` it won't actually `download all available ...` (the action that is decribed)?

Reply:
Correct. I thought that this was clear enough for a boolean action but no strong opinion if you want to change the wording. Maybe simply changing it to `whether to download all available...` is enough.

Reply:
makes sense, Dave, thanks! Probably some minor clarification (e.g. `--no-run-cache` is the default and deprecated) would help. I'll look into that when I have a bit of time.